.azure-pipelines.yml

```
@@ -48,7 +48,7 @@ jobs:
       sudo update-alternatives --set python /usr/bin/python3
       python -m pip install --upgrade pip
       python -m pip install --upgrade wheel
-      python -m pip install --upgrade cmake matplotlib==3.2.2 mpi4py numpy scipy yt==3.6.1
+      python -m pip install --upgrade cmake matplotlib==3.2.2 mpi4py numpy scipy yt
```

We might even be able to relax the matplotlib constrain with yt 4.0 again: :)
```suggestion
      python -m pip install --upgrade cmake matplotlib mpi4py numpy scipy yt
```